test/unit/test_kokonoe.gd

```diff
@@ -213,14 +213,23 @@ func validate_life(p1, l1, p2, l2):
 ##
 ## Tests start here
 ##
+
+func test_kokonoe_boost_pass_gravitron():
+	position_players(player1, 3, player2, 7)
+	assert_eq(player1.get_buddy_location(), -1)
+	give_player_specific_card(player1, "standard_normal_grasp", TestCardId3)
+	assert_true(game_logic.do_boost(player1, TestCardId3))
+	# Don't place gravitron
+	assert_true(game_logic.do_choice(player1, 0))
+	assert_eq(player1.get_buddy_location(), -1)
```

Comment:
can you add advance_turn(player2) just to verify that the turn completed successfully and we're not stuck in some weird decision limbo state. I've been adding that to a lot of tests.